main.py

```diff
@@ -1391,7 +1410,7 @@ def openBooster(userid, username, channel, isWhisper, packname, buying=True):
 
         # alerts
         for w in alertwaifus:
-            threading.Thread(target=sendDrawAlert, args=(channel, w, str(username))).start()
+            threading.Thread(target=sendDrawAlert, args=(channel, w, str(display_name))).start()
```

Comment:
Sneaky display name change here. This may make it very difficult for people to check the puller's hand/profile to try and work out a deal immediately (if needed). I'd suggest changing sendDrawAlert to be like "{display_name} ({username})".

Reply:
This was actually the display name all along. This function never used to pull in the username, but it now has to for the meme pack system. However, adjusting it like that (at least if the names differ by more than case) is probably a good idea regardless.